content/applications/hr/payroll/reporting.rst

```diff
@@ -0,0 +1,205 @@
+:guilabel:`Payroll`, :guilabel:`Meal Vouchers`, :guilabel:`Attachment of Salary`, and
+:guilabel:`DMFA` reports are universal and available for all companies regardless of location, while
+the rest of the reports are specific to Belgium only.
+
+To view a report, go to :menuselection:`Payroll --> Reporting` and click on the specific report to
+view.
+
+.. image:: reporting/reports.png
+   :align: center
+   :alt: Report dashboard view.
+
+If a report is unavailable to a company, a user error will pop up, stating “You must be logged in a
+Belgian company to use this feature.” Click :guilabel:`OK` or the small x in the top right corner to
+close the warning.
+
+The *Filters* and *Favorites* options appear for all reports in the top right. At the top of the
+individual report, click on :guilabel:`Filters` to display the optional filters for the specific
+report. All reports have an option to add a custom filter. Each report will have different default
+custom filter options. Select the parameters, then click the :guilabel:`Apply` smart button.
+
+.. image:: reporting/custom-filter.png
+   :align: center
+   :alt: Report dashboard view.
+
+Clicking on :guilabel:`Favorites` allows for three options: save the current search, add the search
+to the dashboard, or insert the search to a Google spreadsheet.
+
+.. image:: reporting/favorites.png
+   :align: center
+   :alt: Favorite options for reports.
+
+Payroll
+-------
+
+Clicking on :menuselection:`Payroll --> Reporting --> Payroll` will display the *Payroll Analysis*
+report.
+
+.. image:: reporting/payroll-report.png
+   :align: center
+   :alt: Report dashboard view.
+
+This will display all the payslips generated in the last 365 days. This report can display
+metrics for a variety of parameters. Click the :guilabel:`Measures` box and a drop down will appear
+with all the various options to display. The options available are: # Payslip, Basic Wage, Basic
+Wage for Time Off, Days of Paid Time Off, Days of Unforeseen Absence, Days of Unpaid Time Off, Gross
+Wage, Net Wage, Number of Days, Number of Hours, Work Days, Work Hours, and Count.
+
+.. image:: reporting/measures.png
+   :align: center
+   :alt: Report dashboard view.
+
+Bar chart
+~~~~~~~~~
+
+To display the data in a bar chart, click on the :guilabel:`Bar Chart` icon (first icon)in the menu
+bar.
+
+.. image:: reporting/bar-chart.png
+   :align: center
+   :alt: Report dashboard view.
+
+When this option is selected, an option to display the chart as stacked appears. Click the middle
+:guilabel:`Stacked` icon to view the bar chart in a stacked format (where multiple values appear in
+each column). An option to display the columns in ascending or descending order appears at the end
+of the options.
+
+.. tip::
+   Clicking an option will enable the option. To disable it, click the option again. When the option
+   is enabled it will appear grey, when it is inactive, it will appear white.
```

Comment:
```suggestion
   Clicking an option will enable it. To disable the option, click it again. When the option
   is enabled it will appear grey. When it is inactive, it will appear white.
```